src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputOperatorWrap.java

```diff
@@ -39,8 +39,13 @@ void testAssignment()
             0;
     }
 
+    <T extends Comparable & java.io.Serializable>
```

Comment:
Please put good example near the end of the file to not create differences in test results.

Also this good example has no line break. If I needed a line break around `&`, the line break should come before the `&`. If that will pass, then make that the good example, where the line break is before it.

Reply:
Done, moved the method to end of file.